train-system/src/main/java/hu/bme/mit/train/system/TrainSystem.java

```diff
@@ -25,4 +28,12 @@ public TrainUser getUser() {
 		return user;
 	}
 
+	public void startTrainController() {
+		new Timer().schedule(new TimerTask() {
```

Comment:
Based on the readme, the Timer should work in a fixed rate mode, instead of fixed delay

Reply:
Done in next commit